contracts/protocol/WithdrawalUtils.sol

```diff
@@ -43,14 +43,20 @@ contract WithdrawalUtils is IWithdrawalUtils {
     ) external override {
         require(airnodeOrRelayer != address(0), "Airnode/relayer address zero");
         require(protocolId != 0, "Protocol ID zero");
-        bytes32 withdrawalRequestId = keccak256(
-            abi.encodePacked(
-                block.chainid,
-                address(this),
-                msg.sender,
-                ++sponsorToWithdrawalRequestCount[msg.sender]
-            )
-        );
+        bytes32 withdrawalRequestId;
+        // Will not overflow assuming a sponsor cannot make more than
+        // `type(uint256).max` withdrawal requests in practice
+        unchecked {
+            withdrawalRequestId = keccak256(
+                abi.encodePacked(
+                    block.chainid,
+                    address(this),
+                    msg.sender,
+                    ++sponsorToWithdrawalRequestCount[msg.sender]
+                )
+            );
+        }
+
```

Comment:
Does it really make sense to use `unchecked` in places like this where we are not inside a loop and the gas savings on executing this tx with `unchecked` is less that 200 gas if I'm not mistaken?

Reply:
I think it's even less, I mainly did it for the sake of consistency. Perhaps they can be removed for the request counts.

Reply:
I removed the `unchecked`s that require assumptions (except the timestamp accumulation because it happens in a loop)